### PR TITLE
add `dagster` to apps_groups.conf

### PR DESCRIPTION
**Pull request by andrewm4894**
##### Summary

add dagster which is a popular data orchastration tool similar to airflow

https://github.com/dagster-io/dagster

##### Test Plan

Tested locally and all looks good:

image

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

**Comment by ilyam8** [comment]
Hey, @andrewm4894. Can you show a process tree including the `dagster` process with its parent/child processes using `ps faxu`?

**Comment by andrewm4894** [comment]
```
andrew@WINDOWS-RC4NPI5:~$ netdata -V
netdata v1.47.5
```

```
andrew@WINDOWS-RC4NPI5:~$ ps aux
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root           1  0.4  0.0  22100 13404 ?        Ss   11:22   0:01 /sbin/init
root           2  0.0  0.0   2616  1440 ?        Sl   11:22   0:00 /init
root           6  0.0  0.0   2616   132 ?        Sl   11:22   0:00 plan9 --control-socket 6 --log-level 4 --server-fd 7 --pipe-fd 9 --log-truncate
root          54  0.0  0.1  67248 19992 ?        S<s  11:22   0:00 /usr/lib/systemd/systemd-journald
root          98  0.0  0.0  24356  6376 ?        Ss   11:22   0:00 /usr/lib/systemd/systemd-udevd
root         111  0.0  0.0 152936  2200 ?        Ssl  11:22   0:00 snapfuse /var/lib/snapd/snaps/core20_2434.snap /snap/core20/2434 -o ro,nodev,allow_other,suid
root         112  0.0  0.0 228360  2980 ?        Ssl  11:22   0:00 snapfuse /var/lib/snapd/snaps/code_173.snap /snap/code/173 -o ro,nodev,allow_other,suid
root         125  0.0  0.0 153068   804 ?        Ssl  11:22   0:00 snapfuse /var/lib/snapd/snaps/core22_1663.snap /snap/core22/1663 -o ro,nodev,allow_other,suid
root         126  0.0  0.0 153200  2260 ?        Ssl  11:22   0:00 snapfuse /var/lib/snapd/snaps/htop_4407.snap /snap/htop/4407 -o ro,nodev,allow_other,suid
root         130  0.0  0.0 227700  6740 ?        Ssl  11:22   0:00 snapfuse /var/lib/snapd/snaps/snapd_21759.snap /snap/snapd/21759 -o ro,nodev,allow_other,suid
systemd+     152  0.0  0.0  21452 11932 ?        Ss   11:22   0:00 /usr/lib/systemd/systemd-resolved
systemd+     156  0.0  0.0  91020  6508 ?        Ssl  11:22   0:00 /usr/lib/systemd/systemd-timesyncd
root         198  0.0  0.0   4236  2584 ?        Ss   11:22   0:00 /usr/sbin/cron -f -P
message+     199  0.1  0.0   9724  5344 ?        Ss   11:22   0:00 @dbus-daemon --system --address=systemd: --nofork --nopidfile --systemd-activation --syslog-only
root         212  3.7  0.2 2064788 39184 ?       Ssl  11:22   0:13 /usr/lib/snapd/snapd
root         216  0.0  0.0  17976  8228 ?        Ss   11:22   0:00 /usr/lib/systemd/systemd-logind
root         222  0.0  0.0 1756096 14204 ?       Ssl  11:22   0:00 /usr/libexec/wsl-pro-service -vv
root         247  0.0  0.0   3160  1188 hvc0     Ss+  11:22   0:00 /sbin/agetty -o -p -- \u --noclear --keep-baud - 115200,38400,9600 vt220
root         256  0.0  0.0  50336 10080 ?        Ss   11:22   0:00 /usr/lib/systemd/systemd-journald netdata
syslog       257  0.0  0.0 222508  5300 ?        Ssl  11:22   0:00 /usr/sbin/rsyslogd -n -iNONE
root         264  0.0  0.0   3116  1200 tty1     Ss+  11:22   0:00 /sbin/agetty -o -p -- \u --noclear - linux
netdata      310  2.0  0.4 681560 78160 ?        SNsl 11:22   0:07 /usr/sbin/netdata -P /run/netdata/netdata.pid -D
root         316  0.0  0.1 107000 22636 ?        Ssl  11:22   0:00 /usr/bin/python3 /usr/share/unattended-upgrades/unattended-upgrade-shutdown --wait-for-signal
netdata      360  0.0  0.0  49180 12568 ?        SN   11:22   0:00 spawn-plugins
root         617  0.0  0.0   2624   120 ?        Ss   11:22   0:00 /init
root         620  0.0  0.0   2624   124 ?        S    11:22   0:00 /init
andrew       624  0.0  0.0   6184  5284 pts/0    Ss   11:22   0:00 -bash
root         629  0.0  0.0   6820  4712 pts/1    Ss   11:22   0:00 /bin/login -f
andrew       750  0.0  0.0  20100 11344 ?        Ss   11:22   0:00 /usr/lib/systemd/systemd --user
andrew       751  0.0  0.0  21148  1728 ?        S    11:22   0:00 (sd-pam)
root         782  0.0  0.0 1312508 13044 ?       SNl  11:22   0:00 /usr/libexec/netdata/plugins.d/ebpf.plugin 1
netdata      783  0.3  0.4 1309064 66664 ?       SNl  11:22   0:01 /usr/libexec/netdata/plugins.d/go.d.plugin 1
root         803  0.0  0.0  13500  3224 ?        SN   11:22   0:00 /usr/libexec/netdata/plugins.d/nfacct.plugin 1
netdata      806  0.4  0.0 146328  6716 ?        SNl  11:22   0:01 /usr/libexec/netdata/plugins.d/apps.plugin 1
netdata      814  0.1  0.0   4588  3348 ?        SN   11:22   0:00 bash /usr/libexec/netdata/plugins.d/tc-qos-helper.sh 1
netdata      815  0.0  0.0 128308  6948 ?        SNl  11:22   0:00 /usr/libexec/netdata/plugins.d/network-viewer.plugin 1
netdata      816  0.0  0.0 202564  5836 ?        SNl  11:22   0:00 /usr/libexec/netdata/plugins.d/systemd-journal.plugin 1
netdata      817  0.0  0.0  13456  3084 ?        SN   11:22   0:00 /usr/libexec/netdata/plugins.d/debugfs.plugin 1
andrew       830  0.0  0.0   6072  5120 pts/1    S+   11:22   0:00 -bash
netdata      840  0.0  0.0  13596   744 ?        SN   11:22   0:00 spawn-setns
root        1173  0.0  0.0 153596  2644 ?        Ssl  11:22   0:00 snapfuse /var/lib/snapd/snaps/code_174.snap /snap/code/174 -o ro,nodev,allow_other,suid
root        1838  0.0  0.1 370044 22388 ?        Ssl  11:24   0:00 /usr/libexec/packagekitd
polkitd     1843  0.0  0.0 308160  7132 ?        Ssl  11:24   0:00 /usr/lib/polkit-1/polkitd --no-debug
root        1874  0.0  0.0  14196  6884 pts/0    S+   11:24   0:00 sudo sh -c #!/bin/sh # # This script should be run via curl: #   sh -c "$(curl -fsSL https://raw.githubuser
root        1875  0.0  0.0  14196  1268 pts/2    Ss   11:24   0:00 sudo sh -c #!/bin/sh # # This script should be run via curl: #   sh -c "$(curl -fsSL https://raw.githubuser
root        1876  0.9  0.0  11164  7264 pts/2    S    11:24   0:02 zsh -l
root        2361  0.0  0.0   3496  2324 pts/2    S+   11:26   0:00 make local
root        2362  2.3  0.6 129136 110304 pts/2   S+   11:26   0:01 /home/andrew/repos/anomstack/venv/bin/python3 /home/andrew/repos/anomstack/venv/bin/dagster dev -f anomstac
root        2367  7.0  0.9 1036588 150424 pts/2  Sl+  11:26   0:04 /home/andrew/repos/anomstack/venv/bin/python3 -m dagster_webserver --dagster-log-level info --log-format co
root        2368  5.6  0.8 1436756 137144 pts/2  Sl+  11:26   0:04 /home/andrew/repos/anomstack/venv/bin/python3 -m dagster._daemon run --log-level info --log-format colored
root        2374 14.9  2.8 2342932 460080 pts/2  Sl+  11:26   0:10 /home/andrew/repos/anomstack/venv/bin/python3 -m dagster api grpc --lazy-load-user-code --socket /tmp/tmpbt
root        2388 12.7  2.8 2266076 456352 pts/2  Sl+  11:27   0:08 /home/andrew/repos/anomstack/venv/bin/python3 -m dagster api grpc --lazy-load-user-code --socket /tmp/tmpq3
root        2422  0.0  0.0   2624   124 ?        Ss   11:27   0:00 /init
root        2423  0.0  0.0   2624   128 ?        S    11:27   0:00 /init
andrew      2429  0.0  0.0   6072  5196 pts/3    Ss   11:27   0:00 -bash
root        2547  0.0  0.0  17060 12508 pts/2    S+   11:27   0:00 /home/andrew/repos/anomstack/venv/bin/python3 -c from multiprocessing.resource_tracker import main;main(9)
root        3473 68.2  2.7 1450116 449168 pts/2  Sl+  11:27   0:05 /home/andrew/repos/anomstack/venv/bin/python3 -c from multiprocessing.spawn import spawn_main; spawn_main(t
root        3559  114  1.6 958360 274936 pts/2   Rl+  11:28   0:03 /home/andrew/repos/anomstack/venv/bin/python3 -c from multiprocessing.spawn import spawn_main; spawn_main(t
andrew      3622  0.0  0.0   8280  4148 pts/3    R+   11:28   0:00 ps aux
```

**Comment by ilyam8** [comment]
@andrewm4894 `apps.plugin` has been greatly improved and there is no "other" group anymore (nightly version only for now).

This process will be matched as `dagster` without additional configuration

```
root        2362  2.3  0.6 129136 110304 pts/2   S+   11:26   0:01 /home/andrew/repos/anomstack/venv/bin/python3 /home/andrew/repos/anomstack/venv/bin/dagster dev -f anomstac
```

But to match everything we need `*dagster*`.

File: src/collectors/apps.plugin/apps_groups.conf [hunk 1]
```diff
@@ -87,6 +87,8 @@ vernemq: *beam.smp*vernemq* *start_vernemq* *run_erl*vernemq* *vernemq*epmd*
 erlang: beam.smp
 postfix: *postfix*
 
+dagster: dagster*
```
Comment (ilyam8) [inline src/collectors/apps.plugin/apps_groups.conf:90]:
```suggestion
dagster: *dagster*
```

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**